pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -292,7 +292,7 @@ def run_evaluation(self, test=False):
         # init validation or test progress bar
         # main progress bar will already be closed when testing so initial position is free
         position = 2 * self.process_position + (not test)
-        desc = 'Testing' if test else 'Validating'
+        desc = 'Testing' if test and not self.validation else 'Validating'
```

Comment:
it is not clear to me, wi Y have test and validation, then it states it as validation only?

Reply:
probably `self.validation` will need be renamed to `use_validation_set`. also the issue here is that, when `trainer.test()` is called, the model will be evaluated on test set using test step. there's no option to evaluate it on the validation step is what I'm trying to add on here. or would it be better to just use the validation set and step by default if `trainer.test()` is called and test dataset and step is not defined? I can make the changes